File: capegemini_1/Corejava/src/corejava/Animal.java
package corejava;
public class Animal {
   void sound(){
	 System.out.println("animal sound");   
   }	
}
class Dog extends Animal{
	@Override
	void sound() {
	System.out.println("Dog sound");
	}
}
class Cow extends Animal{
	@Override
	void sound() {
	System.out.println("cow sound");
	}
}

File: php/myproject/test/oop/animal.php
<?php
class Animal{
    public  $name;
    public function __construct($name) {
        $this->name=$name;
    }
    public function get_name() {
        return  $this->name;
    }
}

// $animal=new Animal('cow');
// echo "animal name is ".$animal->get_name();
?>
